netket/sampler/autoreg.py

```diff
@@ -137,6 +138,11 @@ def scan_fun(carry, index):
             dtype=sampler.dtype,
         )
 
+        if config.netket_experimental_sharding:
+            σ = jax.lax.with_sharding_constraint(
```

Comment:
what is this doing? a small comment maybe?

Reply:
This is one of the few clear things 😂